snmp_juniper/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "Juniper Networks",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "snmp_juniper",
+  "metric_prefix": "snmp.",
+  "metric_to_check": [
+    "snmp.jnxDcuStatsPackets",
+    "snmp.jnxVirtualChassisPortInPkts",
+    "snmp.jnxScuStatsPackets"
+  ],
+  "creates_events": false,
+  "short_description": "Collect metrics from your Juniper network devices.",
+  "guid": "62f074dc-8b6b-466d-bca2-175194221546",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Juniper",
```

Comment:
The display name is "Juniper Networks" but elsewhere we use "juniper" alone instead. It feels a bit inconsistent. From @naaltman comment, here https://github.com/DataDog/integrations-core/pull/8284#pullrequestreview-561921651 it seems that the brand (equals vendor ?) is now known as "Juniper Networks".

If the vendor is `Juniper Networks`, the other fields should follow that too ? e.g. "public_title": "Datadog-Juniper Networks", "name": "snmp_juniper_networks", "integration_id": "snmp-juniper-networks", the folder name, etc

Reply:
@gmarabout My comment was a bit more general, if we consider that the name of the vendor is `Juniper Networks`. We should create the tile using `ddev create "Juniper Networks"` to have correct value for other fields too (name, integration_id, etc).

Reply:
Could you elaborate? Why are the values incorrect ?

Reply:
I mean, if we consider that `Juniper Networks` is the correct name, then if I recreate the tile using the latest version of the: `ddev create 'Juniper Networks' --type snmp_tile`

That will generate a tile like this:

https://github.com/DataDog/integrations-core/commit/799a7c6609f2bdf4eba49ae7a5ed42f34dffbecc

This ^ example, `integration_id`, `name`, and folder name are different than the one in this PR.

Reply:
Discussed offline, since `juniper` alone is already used elsewhere, let's keep it `juniper` in most cases and only "Juniper Networks" for display.